orangecontrib/bio/widgets3/OWSetEnrichment.py

```diff
@@ -752,11 +832,8 @@ def item(value=None, tooltip=None, user=None):
                 si.setData(value, Qt.UserRole)
             return si
 
-        model = QStandardItemModel()
-        model.setSortRole(Qt.UserRole)
-        model.setHorizontalHeaderLabels(
-            ["Category", "Term", "Count", "Reference count", "p-value",
-             "FDR", "Enrichment"])
+        model = self.source_model
```

Comment:
This model is only appended to. Running multiple enrichement tasks one after the other results in multiplied rows in the model and view:
<img width="557" alt="screen shot 2018-01-11 at 14 20 15" src="https://user-images.githubusercontent.com/4716745/34827483-997e7abc-f6da-11e7-92c3-99f0b34623b1.png">